src/main/java/aisu/command/DeleteCommand.java

```diff
@@ -0,0 +1,47 @@
+package aisu.command;
+
+import aisu.exception.AisuException;
+import aisu.storage.Storage;
+import aisu.tasklist.TaskList;
+import aisu.ui.Ui;
+import aisu.task.Task;
```

Comment:
Possible to order lexicographically as before
```suggestion
import aisu.exception.AisuException;
import aisu.storage.Storage;
import aisu.task.Task;
import aisu.tasklist.TaskList;
import aisu.ui.Ui;
```